zap/src/main/java/org/zaproxy/zap/model/Tech.java

```diff
@@ -19,6 +19,9 @@
  */
 package org.zaproxy.zap.model;
 
+import java.util.Arrays;
+import java.util.TreeSet;
+import org.apache.logging.log4j.core.util.Assert;
```

Comment:
This should be removed, the codebase should not depend on core Log4j classes unless used specifically for logging.